src/ssl.c

```diff
@@ -1217,12 +1217,20 @@ WOLFSSL_API int wolfSSL_get_SessionTicket(WOLFSSL* ssl,
 
 WOLFSSL_API int wolfSSL_set_SessionTicket(WOLFSSL* ssl, byte* buf, word32 bufSz)
 {
-    if (ssl == NULL || (buf == NULL && bufSz > 0))
+    if (ssl == NULL || (buf == NULL && bufSz > 0) || bufSz > SESSION_TICKET_LEN)
         return BAD_FUNC_ARG;
 
-    if (bufSz > 0)
-        XMEMCPY(ssl->session.ticket, buf, bufSz);
+    /* session ticket should only be size of static buffer. Delete dynamic buffer*/
```

Comment:
Why is this true?  How can guarantee that session tickets will never be greater than the static buffer size?